qt/widgets/instrumentview/inc/MantidQtWidgets/InstrumentView/SimpleWidget.h

```diff
@@ -18,21 +20,21 @@ class ProjectionSurface;
 /**
  * A simple widget for drawing unwrapped instrument images.
  */
-class SimpleWidget : public QWidget {
+class SimpleWidget final : public ISimpleWidget {
```

Comment:
I know you didn't pick this name but now we're refactoring could we think of a better one? `Simple` conveys so little information.

Reply:
@martyngigg Sure. Can we do that in a future PR though? We'd like to change as little as possible until we have some tests in.

Reply:
I've added a comment to #31739 to do this

Reply:
Yeah no problem. Thanks!